### ISSUE TITLE
Does -vidmode always force v-sync?

### ISSUE THREAD

**Issue by BinToss** [Sigmmma/c20#97]
`-vidmode x,y` seems to inherit the refresh rate from the most recent player profile. Enforces V-Sync.
`-vidmode x,y,0` seems to set refresh rate to 60 (not 0).
I didn't have the luxury of time to test these thoroughly.

This information isn't noted in the wiki

**Comment by BinToss** [comment]
Given `./haloce.exe -path "profiles" -vidmode 1920,1080 -console` and the last profile's "Refresh Rate"="120 Hz", "Framerate"="No V-Sync", the framerate was locked to 60fps. dgVoodoo was not present and GPU drivers were set to respect application V-Sync and not throttle framerate. Chimera was also set to not throttle framerate.

The No V-Sync can re-applied/saved to override the vidmode setting.

In short, yes, -vidmode temporarily enabled V-Sync.
